assets/stylesheets/style.scss

```diff
@@ -23,15 +23,11 @@
 	//@import 'shared/welcome'; // welcome messages
 	//@import 'shared/infinite-scroll-end'; // Last page marker once infinite scroll has reached end
 
-	@import 'shipping-services'; // Shipping Services Groups, Group and Entry Component styles
-	@import 'shipping-label'; // Shipping Labels specific styles
-
```

Comment:
Because these files were removed. Instead, I added imports for every new component and "sub-application" at the end of the file ([here](https://github.com/Automattic/woocommerce-services/pull/1051/files/995d9174509fc081ffa6a5d130b4007088fedcaf#diff-71e5f6f03e3a279beddfc328664288eaR282))